spec/models/expert_spec.rb

```diff
@@ -35,6 +35,24 @@
     end
   end
 
+  describe 'associations dependencies' do
+    let(:expert) { create :expert }
+    let(:assistance) { create :assistance }
+    let(:ae) { create :assistance_expert, expert: expert, assistance: assistance }
+
+    before do
+      create :match, assistance_expert: ae
+    end
+
+    context 'when removing an assistance' do
+      it {
+        expect{ expert.assistances = [] }.not_to raise_error
+        expect(expert.assistances).to eq []
+#        expect(ae).to be_destroyed
```

Comment:
@gregoirenovel I’d love your feedback on this. (and the whole PR, if you’d like)

The `ae` activerecord is a `through` association table. When running the app, I do see the object being destroyed and the row being deleted, but in the test the object is not marked as `destroyed?`. My guess would be that ActiveRecord in tests are not _actually_ persisted, and testing for `destroyed?` will just fail. Does it make sense?